site/pages/docs/components/custom/thumbnail.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="docs__section">
+    <div class="docs__header">
+      <h1>Thumbnail</h1>
+      <o-label size="small" text="HTML" />
+    </div>
+
+    <docs-item class="explanation">
+      <div slot="body">
+        <p>Thumbnail is a styled image which can be combined to gallery.</p>
+      </div>
+    </docs-item>
+
+    <!-- Basic -->
+    <docs-item title="Basic" :code="thumbnail_basic" v-if="is_html">
```

Comment:
currently when you pass in an image it just renders it in it's natural size
i would expect a thumbnail component to force a certain size by default + to offer size options

Reply:
Having multiple sizes as an option (modifier) would be a great addition to this.

Reply:
@janperse what i meant was that i don't think this is a thumbnail component
since it doesn't make an image look like a thumbnail

but keeps it's original size